docs/landing-zone-v2/README.md

```diff
@@ -99,7 +99,7 @@ This can be deployed once or multiple times per organization as highlighted in t
 
 ### Client Landing Zone
 
-This section is made up of two packages [client-setup](../../solutions/client-setup/) and [client-landing-zone](../../solutions/client-landing-zone/). The client setup package prepares the Config Controller instances for the new client and creates the required service accounts and namespaces for the client to use.
+This section is made up of two packages [client-setup](../../solutions/client-setup/) and [client-landing-zone](../../solutions/client-landing-zone/). The client setup package prepares the Config Controller instances for the new client and creates the required service accounts and namespaces for the client to use.  Both of these client packages must be installed using the [GitOps](#3-deploy-the-infrastructure-using-gitops) approach
```

Comment:
there's a note in this doc to remove the baked-in root sync:

https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/43c96f4e4a909cccec8c56eb36b090477c07dac9/docs/landing-zone-v2/onboarding-client.md?plain=1#L64

Reply:
Alain, very usefull, i appreciate the dialog.
retrofiting script, thank you